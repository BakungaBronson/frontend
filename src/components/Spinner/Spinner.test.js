import React from 'react';

import { shallow } from 'enzyme';

import Spinner from './index';

describe('Spinner Component', () => {
  it('Renders a Spinner component', () => {

    const SpinnerComponent = shallow(<Spinner />);
    expect(SpinnerComponent.find('.SmallSpinnerWrapper').hasClass('SmallSpinnerWrapper')).toBe(true);
    expect(SpinnerComponent).toMatchSnapshot();

  });
});