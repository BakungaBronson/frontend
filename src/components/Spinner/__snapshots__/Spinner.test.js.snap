// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component Renders a Spinner component 1`] = `ShallowWrapper {}`;
